msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1014,17 +1053,14 @@ private void setupConfiguration(Context context) {
         mPublicClientConfiguration.setOAuth2TokenCache(getOAuth2TokenCache());
     }
 
-    @VisibleForTesting
-    static PublicClientApplicationConfiguration loadConfiguration(@NonNull final Context context,
-                                                                  final int configResourceId) {
-        InputStream configStream = context.getResources().openRawResource(configResourceId);
+    private static PublicClientApplicationConfiguration loadConfiguration(InputStream configStream, boolean useDefaultConfigResourceId) {
```

Comment:
I'd suggest changing the naming of the boolean to isDefaultConfiguration.  In this case you're communicating to the method that's it being used to load the default configuration or not.  You're not instructing the method to use the default configuration.

Reply:
I renamed the variable as suggested in aafd2fb.